docs/features/hardware-offload/dpu-gateway-interface.md

```
@@ -0,0 +1,208 @@
+
+- Must be used in DPU host mode (`--ovnkube-node-mode=dpu-host`)
+- Management port netdev must be specified (`--ovnkube-node-mgmt-port-netdev`)
+- Gateway interface must be set to `derive-from-mgmt-port`
+
+## Error Handling
+
+The system will return an error in the following scenarios:
+
+### No Network Devices Found
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)